keras_contrib/layers/convolutional.py

```diff
@@ -227,3 +227,31 @@ def get_config(self):
 Deconv3D = Deconvolution3D
 get_custom_objects().update({"Deconvolution3D": Deconvolution3D})
 get_custom_objects().update({"Deconv3D": Deconv3D})
+
+
+class SubPixelUpscaling(Layer):
```

Comment:
No it is tensorflow and theano now. It directly calls tensorflow's depth_to_scale method in the tensorflow backend.